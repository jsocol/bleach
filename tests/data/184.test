<div id="118"><div draggable="true" ondragstart="event.dataTransfer.setData('text/plain','malicious code');">
    <h1>Drop me</h1>
</div>
<iframe src="http://www.example.org/dropHere.html"></iframe>//["'`-->]]>]</div>
