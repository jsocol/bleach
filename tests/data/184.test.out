<div id="118"><div>
    <h1>Drop me</h1>
</div>
//["'`--&gt;]]&gt;]</div>
